docs/regional_coverage/index.md

```diff
@@ -1,6 +1,6 @@
 # Regional coverage
 
-This section describes:
+This section describes all the regions covered in TIAM followed by base year information. Base-year provides details regarding energy balance, emissions modelling approach and stocks. This section further describes the methodology for demand projection until 2100 and lastly possibilities of import and export between TIAM regions. :
```

Comment:
Please remove colon at the end of this paragraph.

Reply:
Also done

Reply:
Did you remember to push this change?